adcc/LazyMp.py

```diff
@@ -138,7 +143,9 @@ def t2eri(self, space, contraction):
         }
         return expressions[space + contraction].evaluate()
 
+    # TODO: correct caching with set_t2?
```

Comment:
Or just skip supporting it. To do this properly we would have to extend our equations anyway (some implicit simplifications in the adc equations no longer hold IRC from discussions with Manu. So I suggest we just scratch this (unused) feature for now.

Reply:
I've come up with a solution for this. If the feature is not required, we could instead use `functools.lru_cache`, which I would prefer indeed.

Reply:
With `lru_cache`, there is however no way at all to access the cache. Just a "clear" function can be called.